packages/service/src/service.ts

```diff
@@ -27,7 +27,7 @@ const auth = (req, resp, next) => {
 
 passport.use(
   new LocalStrategy(async function (username, _password, done) {
-    const user = await createUser({ name: username });
+    const user = await createUser({ name: username, password: _password });
```

Comment:
adding `_` to the beginning of a variable tells the linter to ignore it for unused variable checks. We should remove it if we are using this variable.